### ISSUE TITLE
5V fan compatibility

### ISSUE THREAD

**Issue by hunordori** [tjko/fanpico#131]
Hi, Is there an easy way to use with 5V (Noctua) fans? I know most of the PC fans are 12 V.
I have 3-4 Noctua fans in my rack and I want to control them, independently from my Home Assistant.

**Comment by tjko** [comment]
FanPico "power-board" was created to allow running high-current fans. But it also works for running fans that use different voltage (5V to 24V), as long as Fan accepts 5V PWM signal to control fan speed.

https://github.com/tjko/fanpico/tree/main/boards/power-board

(I have some PCBs for these and plan on making handful of "kits" available in coming weeks...)

**Comment by tjko** [comment]
@hunordori,  kits for powerboard should now be available on eBay (direct link can be found 
 here: https://github.com/tjko/fanpico/discussions/12)

**Comment by hunordori** [comment]
@tjko Thank you! I received it yesterday. If I understand supplying 5V to the 8404D is not a good idea. I should do 12V for the board and a separate 5V with the "power-board".

**Comment by tjko** [comment]
@hunordori, yes FanPico board was designed to take 12V (it likely won't function correctly at 5V).  You can connect the 5V power supply for the fans into the power input (J1) on the 'power-board'.

Then FAN output from FanPico would be connected to FANCTRL connector on the power-board and the fan would be connected to the corresponding FAN connector on the power-board. 

```
Fan (5V) <-----> [FAN1]PowerBoard[FANCTRL1] <-----> [FAN1]FanPico
```

**Comment by hunordori** [comment]
https://github.com/hunordori/rackpico

Thank you again for the support! I created a 1U case for the board. It is installed in my home lab rack.

**Comment by tjko** [comment]
@hunordori, I added link to your case in the Wiki page. Thanks!

https://github.com/tjko/fanpico/wiki/FanPico-Enclosures-(3D%E2%80%90Printed)
